lib/miq_automation_engine/service_models/miq_ae_service_miq_request_task.rb

```diff
@@ -20,6 +20,14 @@ def finished(msg)
       object_send(:update_and_notify_parent, :state => 'finished', :message => msg)
     end
 
+    def set_canceling
+      object_send(:update_attributes, :cancelation_status => 'canceling')
```

Comment:
@hsong-rh @bzwei Is this method still needed and if so it should be changed to call the backend `cancel` method.

https://github.com/ManageIQ/manageiq/pull/17825/files#diff-eb7db57808b5f20ff8f0f07bfd565890L105
But something still has to change, either the cancel method needs to be defined at the root level, maybe raising NotImplemented or this definition needs to move to the service_template_transformation_plan_task service model.

Also, I would expect this to just use the expose syntax above:
`expose :cancel`